lib/config.js

```diff
@@ -26,17 +26,17 @@ const Config = {
     autocompleteSuggestionPriority: {
       title: "Autocomple Suggestion Priority",
       description:
-        "Specify the sort order of Hydrogen's autocomplete suggestions.\nNote the default providers like snippets have priority of 1.",
+        "Specify the sort order of Hydrogen's autocomplete suggestions. Note the default providers like snippets have priority of `1`. Requires Atom restart to take an effect.",
```

Comment:
Requiring an atom restart is not true. It **only requires** autocomplete-plus restart **or** hydrogen restart. Restarting atom does both of those, but is not required.

Reply:
Is it possible to _restart_ a package ?
I suspect the cached loaded code or something like that remains, non ?

Reply:
enable and disable of a package trigger the services api. In other words our provider is recreated every time one of the packages is enable. You can easily test this by going to package settings and disabling and re-enabling

Reply:
also on disable, the service api cleans up our provider

Reply:
Okay then it would be more precise.
But do you think the concept of "restarting autocomplete-plus" or "restarting Hydrogen" is clear to users ? I mean, to say like "require to disable and enable the package" can sound like weird.
If you so, I'm okay to change the description.

Reply:
I know there is no restart button, but restart is equivalent to off and back on again and disable is equivalent to off and enable is equivalent to on. I think native english speakers would understand restart Hydrogen or Autocomplete-Plus, but I am unsure of language barriers

Reply:
Hmm I don't feel something weird in "restart" word itself here, but rather feel like the concept of "restart" in Atom package can be ambiguous.

Reply:
Seems like reload is better

Reply:
I just would like to hear you suggestion of the alternative to

> Requires Atom restart to take an effect.

Reply:
"Requires a Hydrogen restart to take effect"

Reply:
"Requires a Hydrogen restart to take effect"

Reply:
Let's go just with "restart" allowing general interpretations for users :)